### ISSUE TITLE
v2 alpha versions > alpha9 are considered older by go get

### ISSUE THREAD

**Issue by marco-m** [alecthomas/participle#248]
I noticed the following:

```
$ go get github.com/alecthomas/participle/v2@latest
go: added github.com/alecthomas/participle/v2 v2.0.0-alpha9

$ go get github.com/alecthomas/participle/v2@v2.0.0-alpha11
go: downgraded github.com/alecthomas/participle/v2 v2.0.0-alpha9 => v2.0.0-alpha11
```

This might also explain part of the confusion in #246.

**Comment by alecthomas** [comment]
Ah interesting, you may be right. That seems like a bug in the Go toolchain...

**Comment by alecthomas** [comment]
Though maybe it should be `alpha.1` to `alpha.11`... semver seems to [indicate this](https://semver.org/#spec-item-11). So I think this was my bad. I will try and correct the tags.

**Comment by alecthomas** [comment]
I released v2.0.0-beta.1 which seems to fix this.
